tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -229,3 +229,30 @@ def test_format_name(self):
             )
             == "Joe Michael Bloggs"
         )
+
+    def test_email_link(self):
+        assert (
+            self.transforms.email_link("test@email.com", "test subject", "12345")
+            == '<a href="mailto:test@email.com?subject=test%20subject%2012345">test@email.com</a>'
+        )
+
+    def test_email_link_without_subject_reference(self):
+
+        assert (
+            self.transforms.email_link("test@email.com", "test subject")
+            == '<a href="mailto:test@email.com?subject=test%20subject">test@email.com</a>'
+        )
+
+    def test_email_link_without_subject(self):
+
+        assert (
+            self.transforms.email_link("test@email.com")
+            == '<a href="mailto:test@email.com">test@email.com</a>'
+        )
```

Comment:
I think it would be better to build the tests up as:
- `test_email_link`
- `test_email_link_with_subject`
- `test_email_link_with_subject_and_reference`